README.md

```diff
@@ -99,32 +98,50 @@ color you like:
 
 ## Directory tracking
 
+The `default-directory` would change when you change your working directory.
+and text in the vterm buffer has a text property `working-directory`.
+
+
+
+vterm enable `compilation-shell-minor-mode` by default.
+even you change your working directory, `next-error` `previous-error` still
+can find out the corresponding source code.
+
+
 For `zsh` put this in your `.zshrc`:
 
 ```zsh
-function chpwd() {
-    print -Pn "\e]51;A$(pwd)\e\\";
+vterm_prompt() {
+      print -Pn "\e]51;A$(pwd)\007"
```

Comment:
Why did you change this to `\007`?

Reply:
just copy and modified from my original zshrc config, I will change it to `\e\\`